trade-finance-manager-ui/component-tests/pageRenderer.ts

```diff
@@ -0,0 +1,11 @@
+import { load } from 'cheerio';
+import { assertions } from './assertions';
+import configureNunjucks from '../server/nunjucks-configuration';
+
+const nunjucks = configureNunjucks({}) as { render: (pageLocation: string, params: object) => string };
```

Comment:
I looked around for any declaration files for the `nunjucks` package, but I couldn't find anything. As a result, putting in this explicit type assertion seems like the best way to do it

Reply:
From a quick google this came up: https://www.npmjs.com/package/@types/nunjucks did you consider this/can you tell if this is accurate and might do we what we want?

Otherwise if we leave this as is, then I think we should at least add a comment explaining this and also maybe linking to wherever would be the most useful place to look if we ever wanted to extend this type to cover more of the returned utilities?

Reply:
Not sure why that didn't appear when I looked up the types, but those are what we need - thanks!